### PR TITLE
update: cit-web readme

### PR DESCRIPTION
**Pull request by zacpez**

### CODE REVIEW

File: cit3.0-web/README.md [hunk 1]
```diff
@@ -1,53 +1,79 @@
...
-criteria.
+This frontend application contains two high-lelvel business objectives: Community Investment Opportunities Tool, and Community Information Tool.
+
+## Community Investment Opportunities Tool
+
+The Community Investment Opportunity Tool is a place where Economic Development Officers (EDO) can go to post possible investment opportunities for their respective communities. This tool includes a user authenticated portion where EDO's can go to see their currently posted investment opportunities as well as post new ones. It also includes a public facing view where possible investors can go to see all the currently available opportunities in the province or narrow down the opportunities on a set of filterable criteria.
```
Comment (HypnoMama) [inline cit3.0-web/README.md:7]: province _and_ narrrow rather than or?

File: cit3.0-web/README.md [hunk 2]
```diff
@@ -56,3 +82,28 @@ Runs ESLint on all files and outputs any warnings and errors to the console.
 ### `yarn lint:fix`
 
 Runs ESLint on all files and automatically fixes any errors found.
+
```
Comment (HypnoMama) [inline cit3.0-web/README.md:85]: Love this section!  Super helpful!  Nice work :-)

**Review by HypnoMama — APPROVED** [review]

File: cit3.0-web/README.md [hunk 3]
```diff
@@ -1,53 +1,79 @@
-# B.C. Community Investment Opportunity Tool
+# Frontend Application
 
-The Community Investment Opportunity Tool is a place where Economic Development Officers (EDO) can go to post possible investment opportunities for their respective communities. This tool includes a user authenticated portion where EDO's can go to see their currently posted investment opportunities as well as post new ones. It also includes a public facing view where possible investors can go to see all the currently available opportunities in the province or narrow down the opportunities on a set of filterable
-criteria.
+This frontend application contains two high-lelvel business objectives: Community Investment Opportunities Tool, and Community Information Tool.
```
Comment (BrendanBeach) [inline cit3.0-web/README.md]: Small typo
```suggestion
This frontend application contains two high-level business objectives: Community Investment Opportunities Tool, and Community Information Tool.
```

File: cit3.0-web/README.md [hunk 4]
```diff
@@ -1,53 +1,79 @@
...
+
+## Community Investment Opportunities Tool
+
+The Community Investment Opportunity Tool is a place where Economic Development Officers (EDO) can go to post possible investment opportunities for their respective communities. This tool includes a user authenticated portion where EDO's can go to see their currently posted investment opportunities as well as post new ones. It also includes a public facing view where possible investors can go to see all the currently available opportunities in the province or narrow down the opportunities on a set of filterable criteria.
+
+## Community Information Tool
+
+Is an emedded PowerBI report in addition to a splash page and web interface. The majority of the functionality is implemented in the PowerBI environment.
```
Comment (BrendanBeach) [inline cit3.0-web/README.md]: Small typo and wording suggestion
```suggestion
This is an embedded PowerBI report in addition to a splash page and web interface. The majority of the functionality is implemented in the PowerBI environment.
```

File: cit3.0-web/README.md (hunk 2 above)
Comment (BrendanBeach) [reply cit3.0-web/README.md:85]: I agree, giving a quick explanation of the folders to give people an idea of what to expect before they even look at any code is a great idea.

File: cit3.0-web/README.md [hunk 5]
```diff
@@ -56,3 +82,28 @@ Runs ESLint on all files and outputs any warnings and errors to the console.
...
+
+`mocks` - scripts to mock objects in unit tests.
+
+`components` - visual components and pages in the application.
+
+`constants` - constants strings and numbers shared across application components.
+
+`contexts` - context like authorization, shared across application components.
+
+`hooks` - components-specific code chunk, matched react life cycles well.
```
Comment (BrendanBeach) [inline cit3.0-web/README.md]: I had to read this over a couple of times, maybe it should be reworded a little bit? I might not be reading it correctly.
```suggestion
`hooks` - component-specific code chunks, intended to match react life cycles.
```

**Review by BrendanBeach — APPROVED** [review]
A couple of wording suggestions, but looks good!

**Pull request merged**